### PR TITLE
[zh-cn]Sync user-guide.md

### PR DESCRIPTION
**Pull request by yanrongshi**
about #35263
Preview:
https://github.com/kubernetes/website/pull/35368/files?short_path=3b1f093#diff-3b1f093bc0b084c7fca39dea5eae88040ac47370b5b5e1d022fc64b74d6599f4

### CODE REVIEW

File: content/zh-cn/docs/concepts/windows/user-guide.md [hunk 1]
```diff
@@ -157,17 +157,17 @@ port 80 of the container directly to the Service.
...
-     访问虚拟的服务 IP（在 `kubectl get services` 下查看）
-   * 服务发现，使用 Kubernetes [默认 DNS 后缀](/zh-cn/docs/concepts/services-networking/dns-pod-service/#services)的服务名称，
-     用 `curl` 访问服务名称
-   * 入站连接，在 Linux 控制平面节点或集群外的机器上执行 `curl` 来访问 NodePort 服务
-   * 出站连接，使用 `kubectl exec`，从 Pod 内部执行 `curl` 访问外部 IP
+     命令进入容器，并在 Pod 之间（以及跨主机，如果你有多个 Windows 节点）相互进行 ping 操作。
+   * Service 到 Pod 的通信，在 Linux 控制平面所在的节点以及独立的 Pod 中执行 `curl`
+     命令来访问虚拟的服务 IP（在 `kubectl get services` 命令下查看）。
+   * 服务发现，执行 `curl` 命令来访问 Kubernetes [默认 DNS 后缀](/zh-cn/docs/concepts/services-networking/dns-pod-service/#services)的
+     服务名称。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/windows/user-guide.md:168]:
```suggestion
   * 服务发现，执行 `curl` 命令来访问带有 Kubernetes
     [默认 DNS 后缀](/zh-cn/docs/concepts/services-networking/dns-pod-service/#services)的服务名称。
```

File: content/zh-cn/docs/concepts/windows/user-guide.md [hunk 2]
```diff
@@ -157,17 +157,17 @@ port 80 of the container directly to the Service.
...
-     访问虚拟的服务 IP（在 `kubectl get services` 下查看）
-   * 服务发现，使用 Kubernetes [默认 DNS 后缀](/zh-cn/docs/concepts/services-networking/dns-pod-service/#services)的服务名称，
-     用 `curl` 访问服务名称
-   * 入站连接，在 Linux 控制平面节点或集群外的机器上执行 `curl` 来访问 NodePort 服务
-   * 出站连接，使用 `kubectl exec`，从 Pod 内部执行 `curl` 访问外部 IP
+     命令进入容器，并在 Pod 之间（以及跨主机，如果你有多个 Windows 节点）相互进行 ping 操作。
+   * Service 到 Pod 的通信，在 Linux 控制平面所在的节点以及独立的 Pod 中执行 `curl`
+     命令来访问虚拟的服务 IP（在 `kubectl get services` 命令下查看）。
+   * 服务发现，执行 `curl` 命令来访问带有 Kubernetes 
+   [默认 DNS 后缀](/zh-cn/docs/concepts/services-networking/dns-pod-service/#services)的服务名称。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/windows/user-guide.md:168]: 请仔细查看 review comments.

```suggestion
     [默认 DNS 后缀](/zh-cn/docs/concepts/services-networking/dns-pod-service/#services)的服务名称。
```

**Review by howieyuen — COMMENTED** [review]
/lgtm

**Pull request merged**